bitsandbytes/cuda_setup/main.py

```diff
@@ -20,14 +20,19 @@
 import os
 import errno
 import torch
+import platform
 from warnings import warn
 
 from pathlib import Path
 from typing import Set, Union
 from .env_vars import get_potentially_lib_path_containing_env_vars
 
-CUDA_RUNTIME_LIB: str = "libcudart.so"
 
+IS_WINDOWS_PLATFORM: bool = (platform.system()=="Windows")
+PATH_COLLECTION_SEPARATOR: str = ":" if not IS_WINDOWS_PLATFORM else ";"
+CUDA_RUNTIME_LIB: str = "libcudart.so" if not IS_WINDOWS_PLATFORM else "cudart64_110.dll"
+CUDA_SHARED_LIB_NAME: str = "libcuda.so" if not IS_WINDOWS_PLATFORM else "C:\\Windows\\System32\\nvcuda.dll"
```

Comment:
Yes, it's in System32 - to check if I can just use nvcuda.dll, this looks overkill